src/mscorlib/src/System/Array.cs

```diff
@@ -328,7 +328,7 @@ public unsafe Object GetValue(params int[] indices)
             Contract.EndContractBlock();
 
             TypedReference elemref = new TypedReference();
-            fixed(int* pIndices = indices)
+            fixed(int* pIndices = &indices[0])
```

Comment:
I had the same question as I was considering this change... I dug through the code and it doesn't look like a Rank of 0 is possible:

 1. https://github.com/dotnet/coreclr/blob/a9c90ffd84e987a1793a1ed5c5ad7d89b27d493a/src/vm/ecalllist.h#L1523
 2. https://github.com/dotnet/coreclr/blob/c10c1ff8e3237689212606c9aa5153beec8a1778/src/classlibnative/bcltype/arraynative.cpp#L20-L31
 3. https://github.com/dotnet/coreclr/blob/32f0f9721afb584b4a14d69135bea7ddc129f755/src/vm/object.inl#L163-L167
 4. https://github.com/dotnet/coreclr/blob/32f0f9721afb584b4a14d69135bea7ddc129f755/src/vm/methodtable.inl#L604-L613
 5. https://github.com/dotnet/coreclr/blob/dea1b9e5c0393daa420831eb74177b1fd6546181/src/vm/class.h#L2474-L2485
 6. https://github.com/dotnet/coreclr/blob/32f0f9721afb584b4a14d69135bea7ddc129f755/src/vm/clsload.cpp#L3677-L3679
 7. https://github.com/dotnet/coreclr/blob/32f0f9721afb584b4a14d69135bea7ddc129f755/src/vm/array.cpp#L250